setup.py

```diff
@@ -19,7 +19,7 @@
         "service-identity>=14.0.0",
         "treq>=15.0.0",
         "pyOpenSSL>=0.15.1",
-        "txaio==2.1.0",
+        "txaio==2.2.0",
```

Comment:
Why the bump here? I froze it at 2.1.0 because on linux it threw an error using 2.2.0, I'll re-test.